kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/ExecutorServiceBase.java

```diff
@@ -73,6 +73,11 @@ public String scheduleRequest(String containerId, String payload, String marshal
             jobRequest = marshallerHelper.unmarshal(payload, marshallingType, JobRequestInstance.class);
         }
         Long requestId = null;
+
+        if(!validateCommand(jobRequest.getCommand())){
+            throw new IllegalArgumentException("Invalid command type");
```

Comment:
can we add class (jobRequest.getCommand()) that was not found to provide direct feedback?